### PR TITLE
Toolkit learn page

### PR DESCRIPTION
**Pull request by gurtatiLND**
# What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Styling
- [ ] Build
- [ ] Chore

## Description

- Implement a “Learn More” page for the Toolkit tab(take all info from chatGPT analysis, not from Figma) to provide users with detailed explanations. Include navigation to the “Categories” page for filtering and managing toolkit items effectively.

## Screenshots
Screenshot 2024-12-11 at 13 39 20

Screenshot 2024-12-11 at 13 39 08

Screenshot 2024-12-11 at 13 39 32

## UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_

- [ ] Semantic HTML implemented?
- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Added/updated tests?

_Please aim to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## Delete branch after merge?

- [x] Yes
- [ ] No

## What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by maxitect — CHANGES_REQUESTED** [review]
Thanks Tanya, good work getting the content in!

The style of the info page is not consistent with the one created for moods (particularly around the header component and heading sizes), see screenshots below

Screen Shot 2024-12-11 at 18 01 16

Screen Shot 2024-12-11 at 18 01 16

Perhaps use the moods page styling classes as a guide so we are more consistent

File: src/app/home/page.tsx
Comment (maxitect) [inline src/app/home/page.tsx:1]: I don't think this file is required for this PR, and maybe even for the project? Pending decision to start the app on toolkit, home is just the "/" route and so it is the `page.tsx` directly under `app/`

File: src/app/toolkit/info/CategoriesInfoPage.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          <p className="text-gray-400 mt-3 mb-2">
+            Filter by categories you might find helpful or add your own:
+          </p>
+
+          {/* Scrollable Categories */}
+          <div className={categoriesBarClass}>
+                <div className="flex overflow-x-auto whitespace-nowrap space-x-2">
+                {categories.map((category) => (
+                    <Button
+                    key={category.id} // Use a unique key for each item in the map
```
Comment (maxitect) [inline src/app/toolkit/info/CategoriesInfoPage.tsx:48]:
```suggestion
                    key={category.id}
```

File: src/app/toolkit/info/page.tsx [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+  return (
+    <>
+      {/* Header */}
+      <Header title="Learn More" isInfoPage={true} />
+      <div className="bg-twd-background text-white min-h-screen px-10 py-6">
+      {/* Introduction Section */}
+      <div className="mt-6">
+        <h1 className="text-2xl font-bold text-white mb-5">
+          What is this Tab About?
+        </h1>
```
Comment (maxitect) [inline src/app/toolkit/info/page.tsx:24]:
```suggestion
        <h2 className="text-xl font-bold text-white mb-5">
          What is this Tab About?
        </h2>
```
H1 is in header component, text xl to match other learn more pages

**Review by maxitect — CHANGES_REQUESTED** [review]
Question around home/page.tsx file and accessibility/style of headings (hopefully these can just be committed from here as I added suggestions)

File: src/app/home/page.tsx
Comment (gurtatiLND) [reply src/app/home/page.tsx:1]: Oh, that was a mistake, thanks for noticing!! deleted it.

**Review by maxitect — APPROVED** [review]
Thanks for applying those changes!

**Pull request merged**